### PR TITLE
ISPN-6780 View and filter by cache status in cache container view

### PR DESCRIPTION
**Pull request by vblagoje**
Master only. Rebalancing and split brain are implemented. Framework to implemented any other async status filter is in place. We have to wait for cache indexing status implementation.  Please review but do not integrate yet @ryanemerson

### CODE REVIEW

File: src/main/webapp/cluster-view/cluster-view.controller.js [hunk 1]
```diff
@@ -488,6 +488,54 @@ var app = angular.module('managementConsole')
...
+          if (checkboxes.rebalancing) {
+            if (cache.name in cachedIsRebalancing) {
+              var rebalancing = typeof cachedIsRebalancing[cache.name] === 'boolean' ? cachedIsRebalancing[cache.name] : undefined;
+              if (rebalancing) {
+                caches.push(cache);
+              }
+            } else {
+              modelController.getServer().fetchCacheStats(scope.currentCluster, cache).then(function (response) {
+                var stats = response[0];
+                cachedIsRebalancing[input] = stats['cache-rebalancing-status'] === 'IN_PROGRESS' ? true : false;
```
Comment (ryanemerson) [inline src/main/webapp/cluster-view/cluster-view.controller.js]: @vblagoje `input` is undefined. Is this meant to be `cachedIsRebalancing[cache.name] = ...` ?

**Comment by ryanemerson** [comment]
@vblagoje One other trivial point, should we set [this](https://github.com/vblagoje/infinispan-management-console/blob/8bb6ef13d5a9f4a52016c699b8f1d519cc14751a/src/main/webapp/cluster-view/cluster-view.controller.js#L247) to false to remain consistent with traits and types, both of which start in a non-collapsed state.


Do we have a JIRA for the indexing changes? It makes sense to keep everything linked up

**Comment by vblagoje** [comment]
Yep @ryanemerson I did and I implemented online/offline as well. So now we just wait for https://issues.jboss.org/browse/ISPN-6340 to be completed and we'll wire in the call at that time. Please have another look and if satisfied - integrate it!

**Comment by ryanemerson** [comment]
@vblagoje Integrated https://github.com/infinispan/infinispan-management-console/commit/f10120290a538d9405a0c731c1887cff478483fb, thanks

**Pull request closed without merging**